packages/toolpad-app/src/toolpad/Home.tsx

```diff
@@ -95,9 +116,28 @@ function CreateAppDialog({ onClose, ...props }: CreateAppDialogProps) {
                 setName(event.target.value);
               }}
             />
+
+            <FormControl sx={{ my: 1 }} fullWidth>
```

Comment:
Did you know you can turn a `TextField` into a `Select` using `<TextField select>`? It will take care of the `InputLabel` and `FormControl`.

Reply:
oh... i'll use that then